common/src/main/java/tachyon/network/protocol/RPCBlockReadRequest.java

```diff
@@ -77,14 +84,29 @@ public String toString() {
     return "RPCBlockReadRequest(" + mBlockId + ", " + mOffset + ", " + mLength + ")";
   }
 
+  /**
+   * Gets the id of the block.
```

Comment:
To be concise, if the doc is the same as the return field, then simply having the return field is enough. For example,

```
/**
 * @return the id of the block
 */
```